tests/integration-tests-topologies/src/main/java/org/apache/pulsar/tests/topologies/PulsarClusterSpec.java

```diff
@@ -90,13 +90,14 @@
      *
      * @return the list of external services to start with the cluster.
      */
-    Map<String, GenericContainer<?>> externalServices = Maps.newHashMap();
+    Map<String, GenericContainer<?>> externalServices = Collections.EMPTY_MAP;
```

Comment:
better not to mix style changes with this PR.

Reply:
you are taking a dependency from testng in non testcode, that's not correct.

Reply:
sure. I might be using the wrong class. however the point is don't mixing fixes into one PR. if this is wrong, it is ideally to be fixed in a separate PR.